### ISSUE TITLE
FR: edit and sort favorites

### ISSUE THREAD

**Issue by JustCarmen** [fisharebest/webtrees#4204]
It would be nice to have an option to edit and sort favorites.

**Comment by hartenthaler** [comment]
yes, I'm missing this, too. The only possibility is to edit the database itself.

**Comment by SeniorSupportEngineer** [comment]
That would be nice feature.

**Comment by hartenthaler** [comment]
The table in the database is wt_favorite.

**Comment by hartenthaler** [comment]
There is a custom module related to favorites: https://github.com/bwong789/webtrees-favorites-menu
But the focus of this module is to offer a new menu and to group the favorites of an user.

**Comment by webtrees-pesz** [comment]
See also https://www.webtrees.net/index.php/forum/4-customising/40436-order-favorites#114769
